### PR TITLE
Consistent event pages in WebXR docs

### PR DESCRIPTION
**Pull request by Elchi3**
Attempt to make things somewhat consistent. The discussion is still ongoing in https://github.com/mdn/content/discussions/10544 but this brings the event pages I've touched so far to a consistent state.

### CODE REVIEW

File: files/en-us/web/api/xrreferencespace/reset_event/index.md [hunk 1]
```diff
@@ -41,9 +41,18 @@ onreset = (event) => { });
...
 
-## Usage notes
+## Event properties
+
+_In addition to the properties listed below, properties from the parent interface, {{domxref("Event")}}, are available._
+
+- {{domxref("XRReferenceSpaceEvent.referenceSpace", "referenceSpace")}} {{ReadOnlyInline}}
+  - : An {{domxref("XRReferenceSpace")}} indicating the reference space that generated the event.
+- {{domxref("XRReferenceSpaceEvent.transform", "transform")}} {{ReadOnlyInline}}
+  - : An {{domxref("XRRigidTransform")}} object indicating the position and orientation of the specified `referenceSpace`'s native origin after the event, defined relative to the coordinate system before the event.
```
Comment (jpmedley) [inline files/en-us/web/api/xrreferencespace/reset_event/index.md:53]:
```suggestion
  - : An {{domxref("XRRigidTransform")}} object indicating the position and orientation of the specified `referenceSpace` object's native origin after the event, defined relative to the coordinate system before the event.
```

Comment (jpmedley) [reply files/en-us/web/api/xrreferencespace/reset_event/index.md:53]: I totally admit this is a judgement call. I don't like to mix fonts in the same word.

File: files/en-us/web/api/xrcubelayer/redraw_event/index.md [hunk 2]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRCubeLayer` object when the underlying resou
...
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener("redraw", (event) => { });
+
+onredraw = (event) => { });
+```
+## Event type
```
Comment (jpmedley) [inline files/en-us/web/api/xrcubelayer/redraw_event/index.md:30]:
```suggestion
## Event Type
```

Comment (jpmedley) [reply files/en-us/web/api/xrcubelayer/redraw_event/index.md:30]: I didn't think MDN headings were mixed case. Ignore this if that's changed (and let me know). Can you please also look at the headings not directly covered by your changes.

File: files/en-us/web/api/xrcubelayer/redraw_event/index.md [hunk 3]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRCubeLayer` object when the underlying resou
...
+
+An {{domxref("XRLayerEvent")}}. Inherits from {{domxref("Event")}}.
+
+## Event properties
```
Comment (jpmedley) [inline files/en-us/web/api/xrcubelayer/redraw_event/index.md:34]:
```suggestion
## Event Properties
```

File: files/en-us/web/api/xrcylinderlayer/redraw_event/index.md [hunk 4]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRCylinderLayer` object when the underlying r
...
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener("redraw", (event) => { });
+
+onredraw = (event) => { });
+```
+## Event type
```
Comment (jpmedley) [inline files/en-us/web/api/xrcylinderlayer/redraw_event/index.md:30]:
```suggestion
## Event Type
```

File: files/en-us/web/api/xrcylinderlayer/redraw_event/index.md [hunk 5]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRCylinderLayer` object when the underlying r
...
+
+An {{domxref("XRLayerEvent")}}. Inherits from {{domxref("Event")}}.
+
+## Event properties
```
Comment (jpmedley) [inline files/en-us/web/api/xrcylinderlayer/redraw_event/index.md:34]:
```suggestion
## Event Properties
```

File: files/en-us/web/api/xrequirectlayer/redraw_event/index.md [hunk 6]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XREquirectLayer` object when the underlying r
...
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener("redraw", (event) => { });
+
+onredraw = (event) => { });
+```
+## Event type
```
Comment (jpmedley) [inline files/en-us/web/api/xrequirectlayer/redraw_event/index.md:30]:
```suggestion
## Event Type
```

File: files/en-us/web/api/xrequirectlayer/redraw_event/index.md [hunk 7]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XREquirectLayer` object when the underlying r
...
+
+An {{domxref("XRLayerEvent")}}. Inherits from {{domxref("Event")}}.
+
+## Event properties
```
Comment (jpmedley) [inline files/en-us/web/api/xrequirectlayer/redraw_event/index.md:34]:
```suggestion
## Event Properties
```

File: files/en-us/web/api/xrquadlayer/redraw_event/index.md [hunk 8]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRQuadLayer` object when the underlying resou
...
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener("redraw", (event) => { });
+
+onredraw = (event) => { });
+```
+## Event type
```
Comment (jpmedley) [inline files/en-us/web/api/xrquadlayer/redraw_event/index.md:30]:
```suggestion
## Event Type
```

File: files/en-us/web/api/xrquadlayer/redraw_event/index.md [hunk 9]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRQuadLayer` object when the underlying resou
...
+
+An {{domxref("XRLayerEvent")}}. Inherits from {{domxref("Event")}}.
+
+## Event properties
```
Comment (jpmedley) [inline files/en-us/web/api/xrquadlayer/redraw_event/index.md:34]:
```suggestion
## Event Properties
```

File: files/en-us/web/api/xrsystem/devicechange_event/index.md [hunk 10]
```diff
@@ -20,16 +20,26 @@ A **`devicechange`** event is fired on an {{DOMxRef("XRSystem")}} object whene
...
 
-Triggered whenever the availability of immersive XR devices has changed. For example, when a VR headset or AR goggles have been connected or disconnected.
+```js
+addEventListener('devicechange', (event) => { });
+
+ondevicechange = event => { });
+```
 
-### Event type
+## Event type
```
Comment (jpmedley) [inline files/en-us/web/api/xrsystem/devicechange_event/index.md:33]:
```suggestion
## Event Type
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Generally, this looks really clean. The changes are just nits.

File: files/en-us/web/api/xrcubelayer/redraw_event/index.md [hunk 11]
```diff
@@ -18,28 +18,25 @@ The `redraw` event is sent to the `XRCubeLayer` object when the underlying resou
...
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener("redraw", (event) => { });
+
+onredraw = (event) => { });
+```
+## Event type
+
+An {{domxref("XRLayerEvent")}}. Inherits from {{domxref("Event")}}.
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrcubelayer/redraw_event/index.md:32]: I'm quite unsure about this proposal. I think event type pages are different from event interface pages (e.g. https://developer.mozilla.org/en-US/docs/Web/API/XRLayerEvent which could have the diagram).

If you feel strongly about this, I encourage you to propose this in the discussion on the structure of the events page: https://github.com/mdn/content/discussions/10544

File: files/en-us/web/api/xrcubelayer/redraw_event/index.md (hunk 2 above)
Comment (Elchi3) [reply files/en-us/web/api/xrcubelayer/redraw_event/index.md:30]: Headings on MDN are mixed case.

File: files/en-us/web/api/xrcubelayer/redraw_event/index.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/xrcubelayer/redraw_event/index.md:32]: I plan to. 

I don't understand your issue. The macro doesn't say event or or interface. It just shows the relationship and provides links and does so with automation. To provide this manually I have to look it up. Further more, if an inheritance sequence is expanding by adding upstream items, no one has to remember to update event pages. (This happens sometimes. Remember the audio classes that got base classes a few years ago?)

[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrcubelayer/redraw_event/index.md:32]: OK. I've opened https://github.com/mdn/content/pull/11025 to exercise this.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**